Alignment/OfflineValidation/data/lumiperFullRun2_delivered.txt

```diff
@@ -0,0 +1,1914 @@
+272930	3.076

```

Comment:
I have removed this file again as it was an accidental push. When I was testing the implementation I had converted lumiperFullrun2_delivered.csv to a txt for convenience and forgot to remove it again. I have set the default lumi file to lumiPerRun_Run2.txt which also appears to be what Jussi is using by default for the JetHT validation. I have set it to this for the sake of consistency for now but I think it needs to be clarified what lumi file to set as default in general. My guess is that it would be lumiperFullrun2_delivered.csv since I think this file was used for the paper trends?